src/exports/roo-code.d.ts

```diff
@@ -220,6 +220,7 @@ export type GlobalStateKey =
 	| "remoteBrowserEnabled"
 	| "fireworksModelId"
 	| "fireworksModelInfo"
+	| "firstInstallCompleted" // Flag to track if the extension has been opened after installation
```

Comment:
Isn't this redundant with the `showWelcome` boolean that basically means the same?

Reply:
`showWelcome` is a little too aggressive I'm afraid:
[setShowWelcome(!checkExistKey(newState.apiConfiguration))](https://github.com/Kilo-Org/kilocode/blob/3f1406794fbf1e46a2dc6809a06fd2dd26386e05/webview-ui/src/context/ExtensionStateContext.tsx#L169)